src/aggregation.ts

```diff
@@ -8,6 +8,7 @@ export interface AggregationData {
 }
 
 export function findFirstAggregration(command: Command): AggregationData | null {
+  // eslint-disable-next-line guard-for-in
   for (const key in command) {
```

Comment:
I don't really get this loop on command keys, when `getAggregation` only awaits constant `name` string value.

Reply:
There is problem with the types in this concept, pls check out my comment on `Command`